src/js/streams/CosmosPackagesStream.js

```diff
@@ -0,0 +1,24 @@
+// to be replaced with https://github.com/dcos/dcos-ui/pull/2840
+import { Observable, Subject } from "rxjs";
+
+import CosmosPackagesStore from "#SRC/js/stores/CosmosPackagesStore";
+
+const cosmosPackageSource = Observable.timer(1000, 2000)
+  .map(() => CosmosPackagesStore.fetchRepositories())
+  .map(() => CosmosPackagesStore.getRepositories())
+  .distinctUntilChanged();
+
+const searchPackages = new Subject();
+const searchTerms = searchPackages
+  .merge(Observable.of(""))
```

Comment:
What does `Observable.of("")` mean?

Reply:
`Observable.of("")` creates an observable of a single value, emits it, and completes.

The `.merge(Observable.of(""))` could be more clearly written as startWith(""). It is a way to give a first value for a stream. 

Will fix that.